Makefile

```diff
@@ -180,6 +180,10 @@ killall:
 
 clean:
 	@./rebar3 clean
+	( cd apps/aecuckoo && $(MAKE) clean; )
```

Comment:
Actually this is already meant to be run by rebar pre_hook in apps/aecuckoo/rebar/config